### DISCUSSION TITLE
High Availability using multiple Standalone CA/RA/VA installations.

### DISCUSSION THREAD

**Discussion by bsjaramillo** [Keyfactor/ejbca-ce#320] [General]
I was wondering if it is a good idea to use multiple single installations (using the community version) to create a high-availability service with load balancers. It seems a bit strange to me to replicate the complete installation and configuration of the PKI on each server designated for the load balancer.

For example, I attach the following image.
![image](https://github.com/Keyfactor/ejbca-ce/assets/33032851/355d454e-f595-4374-b527-1587c3051c3f)

**Comment by primetomas** [comment]
Yes it's a good idea, but you don't "install" each node as they share the same database. See documentation here:
https://doc.primekey.com/ejbca/ejbca-installation/high-availability-and-clustering
